web3-dev/smart-contracts/sc-execution.rst

```diff
@@ -0,0 +1,66 @@
+========================
+Smart contract execution
+========================
+
+Introduction
+============
+
+In Massa you can execute arbitrary smart contracts, you are not limited to deploy and then call the smart contract.
+There is different possibilities to execute smart contracts in different contexts.
+This documentation will explain you the different possibilities and how to use them.
+You can choose the one that match the most your usecase.
+
+There is examples for the differents ways to execute smart contracts available
```

Comment:
```suggestion
There are examples for the different ways to execute smart contracts available
```

Reply:
You can find examples for the different ways to execute smart contracts